### DISCUSSION TITLE
git@github.com: Permission denied (publickey)

### DISCUSSION THREAD

**Discussion by Nariouss** [Almamu/linux-wallpaperengine#215] [Q&A]
hello, I have problems with your linux version of wallpaper engine, right after I put "git clone git@github.com:Almamu/linux-wallpaperengine.git" I have a error message:

nariouss@nariouss:~$ git clone git@github.com:Almamu/linux-wallpaperengine.git
Cloning into 'linux-wallpaperengine'...
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
nariouss@nariouss:~$ 

Thank you for taking time for helping me to fix this error.

**Comment by Almamu** [comment]
Try with the https link to clone the repo instead: https://github.com/Almamu/linux-wallpaperengine.git

**Reply by Nariouss** [reply]
Hello, thank you, I just checked the files (if wallpaper engine was downladed) and if everything was good on steam (proton compatibility) and it seems like everything is ok, the game is downloaded on my main SSD, and I just tryed to download via the HTTPS method, and I recived this error:

nariouss@nariouss:~$ https://github.com/Almamu/linux-wallpaperengine.git
bash: https://github.com/Almamu/linux-wallpaperengine.git: No such file or directory
nariouss@nariouss:~$ 

Thank you.
